clang/include/clang/Basic/Cuda.h

```diff
@@ -163,6 +220,15 @@ static inline bool IsAMDOffloadArch(OffloadArch A) {
   return A >= OffloadArch::GFX600 && A < OffloadArch::Generic;
 }
 
+static inline bool IsIntelCPUArch(OffloadArch Arch) {
```

Comment:
For sake of completion, can you please add IsIntelOffloadArch as well?

Thanks